### ISSUE TITLE
Code Giving Wrong MainNet Wallet Address from Private Key

### ISSUE THREAD

**Issue by prchdev** [griffo-io/tron.net#4]
Hello All,

I am using below code to get wallet address from private key.
`Dim TRONAccount = Tron.Net.Client.WalletAddress.MainNetWalletAddress(Tron.Net.Crypto.ECKey.FromPrivate(Tron.Net.Common.Utils.FromHexToByteArray(Trim(txtPrivateKey.Text))))`

Unfortunately I get wallet address "TKjhq2fPWKKexa8eRSDQnFeWKxyob3fF2n" but when I have imported same key in official TRON Wallet or on Joyso Exchange its giving me "TVsTKpH6vSkugnRe2zwrKtXW3BthRvkXrP" my wallet address.

I have tried to use below code also.
`Dim TRONAccount = Tron.Net.Client.WalletAddress.MainNetWalletAddress(Tron.Net.Crypto.ECKey.FromPrivateHexString(Trim(txtPrivateKey.Text)))`

But same issue exist.

Your help appreciated.

**Comment by dariusdev** [comment]
The same problem for me. When i try to generate private key:

```
 WalletAddress address = Tron.Net.Client.WalletAddress.MainNetWalletAddress();
string privateKey = Tron.Net.Common.Utils.ToHexString(address.Value);
```

When import this private key to Wallet it shows different public key

**Comment by EnjoX** [comment]
I have created a pull request to fix this issue, if you need it see https://github.com/EnjoX/tron.net

**Comment by dariogriffo** [comment]
fixed on 0.7.1
